src/screens/Home/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import logo from 'assets/images/logo.svg';
+import DefaultLayout from 'components/Layout/Default';
+
+const HomeScreen = (): JSX.Element => {
```

Comment:
We can omit the return type here.
```suggestion
const HomeScreen = () => {
```

Reply:
Thank you! Updated at [38f1b30](https://github.com/rosle/react-nimble-survey/pull/11/commits/38f1b30bcab5bed5bb82f7b933a22170e33f3277) 🚀